### PR TITLE
adding README

### LINKED ISSUE #28: Add README for all services

### PR DESCRIPTION
**Pull request by Keyshal-211**
## Description of changes
Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

  
## How has this been tested?

Please describe the test strategy you followed. If not tested, please explain why.

- [ ] automated unit test
- [ ] automated integration test
- [ ] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.
    
## Checklist before requesting a review

- [X] My code is easy to understand
- [x] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] My code fulfills all acceptance criteria
- [ ] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [x] I have made corresponding changes to the documentation or
  the [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I made no breaking changes in the database schema or if so, I will perform a database migration

## Checklist for reviewer

- The code is easy to understand
- The code follows
  the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of
  this project
- The code is tested or if not, the reason is documented or discussed
- The added and existing tests reasonably cover the code change
- The code has no breaking changes in the database schema or if so, the assignee is aware of it and
  will perform a database migration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,126 +1,14 @@
...
-  - *dto*
-  - *exception*
-  - *persistence*
-    - *dao*
-    - *repository*
-    - *mapper*
-  - *service*
-  - *util* (optional, if needed)
-  - *validation*
+Each flashcard embodies a two-sided structure, resembling the classic format. On one side, it presents educational content in various forms, such as text, images, or even videos, along with a thought-provoking question pertaining to the material that students have previously engaged with.
```
Comment (PaulBredl) [inline README.md]: This is wrong, we don't have two sides flashcards...
I don't know if we have a flashcard concept somewhere, but you could check the wiki if there is one and link it here

File: README.md [hunk 2]
```diff
@@ -1,126 +1,14 @@
...
-    - *repository*
-    - *mapper*
-  - *service*
-  - *util* (optional, if needed)
-  - *validation*
+Each flashcard embodies a two-sided structure, resembling the classic format. On one side, it presents educational content in various forms, such as text, images, or even videos, along with a thought-provoking question pertaining to the material that students have previously engaged with. 
+Lecturers manually create these questions to tailor the learning experience.
 
-Detailed description of the packages:
+Flipping to the other side of the virtual flashcard reveals the answer to the posed question. Here, students can gauge their understanding and knowledge retention by assessing whether they knew the answer beforehand or not.
```
Comment (PaulBredl) [inline README.md]: This and the next section are just frontend details. That has nothing todo with this service.
Maybe describe what functionality the API provides?

Comment (Keyshal-211) [reply README.md]: As I said like I followed our old wiki and that's why it becomes wrong, but I will change all services

**Comment by PaulBredl** [comment]
Done with other pull request

**Pull request closed without merging**